spec/clusterable/ht_item_spec.rb

```diff
@@ -60,27 +60,56 @@
     expect(htitem.n_chron).to eq("")
   end
 
-  xit "has an access of deny or allow" do
-    expect(build(:ht_item).access).to be_in(["allow", "deny"])
+  it "has an access of deny or allow" do
+    expect(build(:ht_item).access).to eq("allow").or eq("deny")
   end
 
-  xdescribe "#billing_entity" do
+  describe "#billing_entity" do
     it "is automatically set when collection_code is set" do
       expect(build(:ht_item, collection_code: "KEIO").billing_entity).to eq("hathitrust")
     end
   end
 
-  xdescribe "#to_hash" do
-    it "contains billing_entity" do
-      expect(htitem_hash[:billing_entity]).not_to be nil
+  describe "#cluster" do
+    include_context "with cluster ocns table"
+
+    it "returns a cluster with all its ocns" do
+      create(:cluster, ocns: [1, 2])
+      h = build(:ht_item, ocns: [1, 2])
+
+      expect(h.cluster.ocns).to contain_exactly(1, 2)
+    end
+
+    it "returns a cluster with some of its ocns (if there is only one)" do
+      create(:cluster, ocns: [1])
+      h = build(:ht_item, ocns: [1, 2])
+
+      expect(h.cluster.ocns).to contain_exactly(1)
```

Comment:
I must be confused. I expected `expect(h.cluster.ocns).to contain_exactly(1, 2)` here too.

Reply:
At this point the ht item has been initialized but hasn't changed anything in the database. The point here is that `h.cluster` on its own doesn't change anything.